### DISCUSSION TITLE
Easing functions were removed

### DISCUSSION THREAD

**Discussion by konradjurk** [apexcharts/apexcharts.js#4821] [General]
### Description

The easing functions have been removed 3 weeks ago, without any apparent reason, a mention in the release notes or a migration guide: https://github.com/apexcharts/apexcharts.js/commit/038923eb6163afe658c55247234a4414a3d53d54

Was this committed accidentally? If not, what is the migration path?

Thanks

**Comment by junedchhipa** [comment]
apexcharts depends on svg.js for these animations and with upgrade to svg.js@3.0.0, these easing functions were removed from svg.js.
As apexcharts@4.0.0 is a major release, it is expected that there will be some breaking changes. You're probably right about not including this in the release notes, I will edit those notes.

**Reply by konradjurk** [reply]
Thank you for the quick reply!

**Comment by nolimitdev** [comment]
@junedchhipa svg.js@3.0.0 here https://github.com/svgdotjs/svg.js/blob/master/CHANGELOG.md#300---2018-12-01 does not mention removing easing functions. We are really missing option `chart.easing = "linear"`. Now our linear line real-time charts are broken. Before animation was fluent so it looked great but now it is jerky :(

Now it seems it is behaving like with option `chart.easing = "easeout"`. So really no chance to change? Behaving like `chart.easing = "easeinout"` would be better for us as replacement for `chart.easing = "linear"`.

**Reply by nolimitdev** [reply]
> @junedchhipa svg.js@3.0.0 here https://github.com/svgdotjs/svg.js/blob/master/CHANGELOG.md#300---2018-12-01 does not mention removing easing functions. We are really missing option `chart.easing = "linear"`. Now our linear line real-time charts are broken. Before animation was fluent so it looked great but now it is jerky :(
> 
> Now it seems it is behaving like with option `chart.easing = "easeout"`. So really no chance to change? Behaving like `chart.easing = "easeinout"` would be better for us as replacement for `chart.easing = "linear"`.

@junedchhipa @brianlagunas Could you check it please? It seems that easing is not removed by changelog as mentioned above and also not removed by documentation here: https://svgjs.dev/docs/3.0/animating/#easing Easing is still available in any latest releases:
https://svgjs.dev/docs/3.0/animating/#easing
https://svgjs.dev/docs/3.1/animating/#easing
https://svgjs.dev/docs/3.2/animating/#easing 

Could be please this discussion and/or issue https://github.com/apexcharts/apexcharts.js/issues/4819 re-opened? Thanks!
